Automattic/wp-calypso
# Marketing > Traffic: Remove gray background from 'Related Posts'

Fixes https://github.com/Automattic/wp-calypso/issues/75599

## Proposed Changes

Replaces the gray background on 'Related posts' with a gray border to address accessibility issues with contrast.

### Before

<img width="1092" alt="image" src="https://github.com/Automattic/wp-calypso/assets/36432/1b2f1999-faed-4a16-9678-ae1b0e2e3c12">

### After

<img width="1084" alt="image" src="https://github.com/Automattic/wp-calypso/assets/36432/08e62990-98e3-4c65-bdfe-f7a598e2666a">

## Testing Instructions

1. Navigate to the 'Traffic' tab under 'Marketing'.
2. Run axe DevTools.
3. Verify the 'Related posts' elements no longer appear under the 'Elements must have sufficient color contrast' warning (but the page description will still be flagged).
